frontend/components/common/BasePanel.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+    <modal :visible="true" :mask-closable="false" :closable="false" :footer="null">
+        <!-- Default Base Panel -->
+        <div v-if="state === 'default'">
+            <div class="p-5 gap-5 text-lg font-bold break-words"><slot /></div>
+        </div>
+        <!-- Inactive Base Panel -->
+        <div v-if="state === 'inactive'">
+            <div class="p-5 gap-5 text-lg font-bold text-gray-500 break-words">
+                <slot />
+            </div>
+        </div>
+        <!-- Incorrect Base Panel -->
+        <div v-if="state === 'incorrect'">
+            <div class="p-5 gap-5 text-lg font-bold text-red-500 break-words">
+                <slot />
+            </div>
+            <div class="border-solid border-2 p-5 gap-5">
+                <div class="grid grid-cols-2">
+                    <span>Amount in wallet:</span>
+                    <span>100 DAI</span>
+                </div>
+                <div>lorem ipsum</div>
+                <div class="flex justify-end p-5 gap-5">
+                    <BaseButton type="primary">Click here</BaseButton>
+                </div>
```

Comment:
This should not be part of the `BasePanel`, but should be provided as content (in other words as a `<slot />`)